books/canute.@@__canute_manual__@@.txt

```diff
@@ -0,0 +1,2 @@
+current_page = 1
+bookmarks = []
```

Comment:
All the `books/*.txt` files need to be removed from git and `books/*.txt` needs to be added to .gitignore.